src/views/ViewRoutingSlip.vue

```diff
@@ -14,9 +14,6 @@
           </header>
         </v-col>
 
-        <v-col cols="12">
-          <print-routing-slip />
-        </v-col>
         <v-col cols="12" class="mb-5">
```

Comment:
its not needed?

Reply:
No print we are going to remove, roy mentioned